src/referencePicker/providerHelper.js

```diff
@@ -26,6 +26,9 @@ if (!window._vue_richtext_reference_provider_timestamps) {
  * @returns {Object} The provider object
  */
 export function getProvider(providerId) {
+	if (providerId === 'any-link') {
```

Comment:
Please make this a constant that is also exported :)

Reply:
:+1: